packages/ui/src/components/contact/Contact.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { Contact } from './Contact'
+
+const meta: Meta<typeof Contact> = {
+  component: Contact,
+}
+
+export default meta
+
+type Story = StoryObj<typeof Contact>
+
+export const Default: Story = {
+  // TODO: Fill args
+  args: {
+    address: '',
+    email: '',
+    phone: '',
+    title: '',
```

Comment:
You can merge it but this story args are empty. You could fill this part. We will use the component on the page when the page is ready. For now, we can create components and test them in Storybook.